lib/museum.rb

```diff
@@ -14,17 +14,13 @@ def add_exhibit(exhibit)
     end
 
     def interests_list(patron)
-        patron.interests.map do |interest|
-            interest
-        end
+        patron.interests
```

Comment:
Anytime you map over and simple return the item you're iterating over, you likely don't need the enumerator and you can just return the original collection.